package.json

```diff
@@ -13,20 +13,26 @@
   "license": "MIT",
   "exports": {
     ".": {
+      "node": "./dist/index.js",
       "require": "./dist/index.js",
       "import": "./dist/index.mjs",
       "types": "./dist/index.d.ts"
     },
-    "./manager.js": {
+    "./manager": {
       "require": "./dist/manager.js",
       "import": "./dist/manager.mjs",
       "types": "./dist/manager.d.ts"
     },
-    "./preview.js": {
+    "./preview": {
       "require": "./dist/preview.js",
       "import": "./dist/preview.mjs",
       "types": "./dist/preview.d.ts"
     },
+    "./register": {
```

Comment:
this shouldn't be required?

Reply:
The addon should be registered using the `manager` and `preview`

Reply:
Yeah that's what I was thinking too but for some reason the addon was not showing and this fixed the issue. maybe it was something else that was messing up the build but that's the only change I did between the two versions to make it work. It's not easy to test this as you kind of have to publish an addon on npm. Can you try to publish the current version of the addon on this repo to npm to see if that works for you ?

Reply:
I didn't realize the actual issue seems to be the .js part and not the register thing I updated the PR

Reply:
Cool, I'm going to publish an addon and test it out today before merging this.

Reply:
Perfect 🎉